app/controllers/v1/recipes_controller.rb

```diff
@@ -17,9 +17,18 @@ def create
     end
   end
 
+  def show
+    recipe = Recipe.find(params[:id])
+    render json: recipe, serializer: Recipes::IndexSerializer
```

Comment:
You should use a separate serializer here, please create a `Recipes::ShowSerializer`